packages/sdk/src/swap/sdk.ts

```diff
@@ -494,7 +494,7 @@ export class SwapSDK {
         channelOpeningFee: result.channelOpeningFee,
       };
     } else {
-      response = await this.trpc.openSwapDepositChannel.mutate(depositAddressRequest);
+      response = await this.trpc.openSwapDepositChannel.mutate({ ...depositAddressRequest, quote });
```

Comment:
we don't use the function yet in the swapping app, we should add it to the v1 method as well

Reply:
the v1 function does not accept a quote at the moment. will change the swapping app to use v2 once this is merged 🙂

Reply:
the alternative would be changing the v1 method to additionally accept a quote, but feels wrong as we already have a v2 method that does this